packages/js/onboarding/src/components/WooPaymentsMethodsLogos/PaymentMethodsLogos.scss

```diff
@@ -38,3 +53,21 @@
 		}
 	}
 }
+
+.inside-popover {
+	display: grid;
+	grid-template-columns: repeat(4, 1fr);
+	grid-template-rows: repeat(3, 1fr);
+	gap: 8px;
+	padding: 0px;
+}
+
+.components-popover__content {
+	background: $white;
+	border: 1px solid $gray-100;
+	border-radius: 3px;
+	max-width: 246px;
+	padding: 8px;
```

Comment:
I believe we can use the `$gap-smaller` here and for the `margin-bottom` below as well.